mockito-kotlin/src/main/kotlin/org/mockito/kotlin/OngoingStubbing.kt

```diff
@@ -124,3 +126,15 @@ infix fun <T> OngoingStubbing<T>.doAnswer(answer: Answer<*>): OngoingStubbing<T>
 infix fun <T> OngoingStubbing<T>.doAnswer(answer: (InvocationOnMock) -> T?): OngoingStubbing<T> {
     return thenAnswer(answer)
 }
+
+infix fun <T> OngoingStubbing<T>.willAnswer(answer: suspend (InvocationOnMock) -> T?): OngoingStubbing<T> {
```

Comment:
The `willX` notation is already used in BDD-style mocking. To keep the naming consistent with the other methods defined, WDYT about the following: `doAnswerForCoroutine`?

Reply:
`doAnswerForCoroutine` sounds weird to me, but consistency is a first-class citizen always. Just maybe we should call it `doSuspendableAnswer`? It will sound like `on loadData do suspendable answer`

Reply:
`doSuspendableAnswer` sounds good to me.

Reply:
Final touched done. I leave `doSuspendableAnswer` name